content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -224,29 +137,28 @@ export KUBECONFIG=/etc/kubernetes/admin.conf
 ```
 
 Make a record of the `kubeadm join` command that `kubeadm init` outputs. You
-will need this in a moment.
+need this command to [join nodes to your cluster](#join-nodes).
```

Comment:
OK, but probably you need to save this somewhere protected, like a password manager. You probably don't want people copying it into a text file or storing it on Github, for instance. OK, you cover some of that in the next paragraph, but I think this can be reorganized.